.gitignore

```diff
@@ -5,3 +5,4 @@ public_html/assets
 .phpunit.cache
 yarn-error.log
 .php-cs-fixer.cache
+.DS_Store
```

Comment:
User specific files should be excluded here, this should be set in your global `.gitignore` file.

```bash
git config --global core.excludesfile $HOME/.gitignore
```

and put `.DS_Store` to that file.